src/scripts.js

```diff
@@ -12,49 +14,124 @@ import './images/turing-logo.png'
 //GET ELEMENTS BY ID 
 const customerDashboardView = document.getElementById('customerDashboardView')
 const allRoomBookings = document.getElementById('allRoomBookings')
-const roomBookingDetails = document.getElementById('roomBookingDetails')
+const pastReservationsSection = document.getElementById('pastReservationsSection')
+const upcomingReservationsSection = document.getElementById('upcomingReservationsSection')
+const totalAmountSpentDisplay = document.getElementById('totalAmountSpentDisplay')
+const bookARoomBtn = document.getElementById('bookARoomBtn')
+const bookARoomView = document.getElementById('bookARoomView')
+const dateSelectBtn = document.getElementById('dateSelectBtn')
+const roomsAvailableDisplay = document.getElementById('roomsAvailableDisplay')
+const dropDownButton = document.getElementById('dropDownButton');
+const myDropdown = document.getElementById("myDropdown");
+
 
 let customer;
+let bookings;
 
 
 const loadPage = () => {
   getData()
   .then((data) => {
-    customer = new Customer(data.customers[0]);
+    customer = new Customer(data[0].customers[0]);
+    bookings = new Bookings(data[1].bookings)
     displayCustomerRoomDashboard();
   })
 };
 
 const getData = () => {
-  return Promise.resolve(fetchApiData('customers'))
+  return Promise.all([fetchApiData('customers'), fetchApiData('bookings')])
 };
 
 const displayCustomerRoomDashboard = () => {
   customer.findBookings().then(() => {
-  
-    customer.customerBookings.forEach(elem => {
-      let splitDate = elem.date.split('/') 
-      splitDate.splice(0, 0, splitDate[2]);
-      splitDate.pop();
-      splitDate.splice(0, 0, splitDate[2]);
-      splitDate.pop();
-      elem.date = splitDate.join('/')
+  customer.rearrangeDate();
+  customer.getTotalAmountSpentOnRooms().then(() => {
+    let totalCost = customer.calculateRoomTotal();
+      totalAmountSpentDisplay.innerText = `$${totalCost}`
+
+    customer.pastReservations.forEach((elem) => {
```

Comment:
I love thinking about how this shows up on the page, this is a very cool display idea.